src/backend/joanie/tests/core/test_api_organizations_course_product_relations.py

```diff
@@ -0,0 +1,276 @@
+"""
+Test suite for Organization's CourseProductRelation API endpoint.
+"""
+
+from joanie.core import factories, models
+from joanie.tests.base import BaseAPITestCase
+
+
+class OrganizationCourseProductRelationApiTest(BaseAPITestCase):
```

Comment:
Can you write the following tests please ? 
In this way all viewset cases are battle tested and then we will need to update the api behavior, we will see immediately their impacts.

- `read_list_anonymous`
- `read_list_without_accesses`
- `read_detail_anonymous`
- `read_detail_without_accesses`